drivers/scheduler/k8s/specs/wordpress/aws/aws-storage-class.yaml

```diff
@@ -0,0 +1,17 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: wordpress-sc
+provisioner: kubernetes.io/aws-ebs
+parameters:
+  type: gp2
+  fsType: ext4
+---
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: wordpress-sc-shared
+provisioner: kubernetes.io/aws-ebs
+parameters:
+  type: gp2
+  fsType: ext4
```

Comment:
same comment as nginx. Does this AWS storage class allow ReadWriteMany PVCs?

Reply:
This is a good point. Basically it does not. So I'll change pvc access mode to RWO.

Reply:
The right thing would be to use EFS.

https://containerjournal.com/topics/container-networking/using-ebs-and-efs-as-persistent-volume-in-kubernetes/

Reply:
okay, may I roll back changes where I've changed access modes. I'll use only specific apps for eks test and I'll bring EFS provisioner after that. Installing efs since it is non-standard provisioner can be quite tricky.